src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -25,24 +25,24 @@
 public class TickerTask implements Runnable {
 	
 	private boolean halted = false;
+
+	final Map<Location, Location> move = new HashMap<>();
+	final Map<Location, Boolean> delete = new HashMap<>();
```

Comment:
Please leave these fields as protected.
Using the default access modifier is not recommended.